src/main/java/Duke/utils/Parser.java

```diff
@@ -0,0 +1,35 @@
+package Duke.utils;
+
+import Duke.command.*;
+import Duke.exceptions.DukeException;
+import Duke.task.TaskType;
+
+public class Parser {
+
+    public static Command parse(String command) throws DukeException {
+        if(command.equals("bye")) {
```

Comment:
Should there be a space here? Eg: if[space](command... 
I notice this in some other places as well, feel free to edit! :)